Rentit-frontend/src/pages/login/login.jsx

```diff
@@ -16,13 +17,6 @@ const Login = () => {
   const [loginData, setLoginData] = useState(null);
   const url = 'http://127.0.0.1:2000/api-rentit/v1/login';
```

Comment:
Add `http://127.0.0.1:2000` to `.env` when you deploy, change the domain you won't have to make changes in multiple files.